http-api/src/main/java/net/runelite/http/api/feed/FeedClient.java

```diff
@@ -55,7 +55,7 @@ public FeedResult lookupFeed() throws IOException
 		{
 			if (!response.isSuccessful())
 			{
-				logger.debug("Error looking up feed: {}", response.message());
+				logger.debug("Error looking up feed: {}", response.toString());
```

Comment:
Doesn't the logger call toString() natively on the object passed in? It seems redundant to have a toString() here if that's the case.

Ref: https://github.com/runelite/runelite/issues/8042#issuecomment-468382802

Reply:
It does